docs/configuration.md

```diff
@@ -0,0 +1,313 @@
+# Configuration
+
+The configuration for `jsii` is recoded in the `package.json` file, which is
+the standard package manifest for NPM packages. This document describes the
+constraints and extensions `jsii` adds to the [package.json schema].
+
+[package.json schema]: https://docs.npmjs.com/files/package.json
+
+## Additional Requirements & Extensions
+
+In order to be able to generate valid packages for all the supported target
+package managers, certain fields that are optional in the standard
+[package.json schema] are required by `jsii`.
+
+For example, Maven Central requires packages to carry [sufficient metadata],
+such as *developer information* and *license*, in order to be valid for
+publishing.
+
+Field       | Required | Extensions
+------------|----------|--------------------------------------------------------
+`author`    | Required | `author.organization`
```

Comment:
Add a `Description` column and explain what this field means or link to doc

Reply:
I agree. I was trying to find the best format for this doc but I was spending too much time. I'll revisit later (probably soon).